cloudbees-rollout/README.md

```diff
@@ -0,0 +1,3 @@
+# CloudBees Rollout Examples
```

Comment:
CloudBees Rollout is now CloudBees Feature Flags, so this directory can be deleted.

Reply:
Done.